testfiles/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ add_simulation_test(IDF_FILE 5ZoneSupRetPlenRAB.idf EPW_FILE USA_IL_Chicago-OHar
 add_simulation_test(IDF_FILE 5ZoneSupRetPlenVSATU.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE 5ZoneSwimmingPool.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE 5ZoneSwimmingPoolZoneMultipliers.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+add_simulation_test(IDF_FILE 5ZoneSystemNodeReset.idf EPW_FILE USA_IL_Chicago-Ohare.Intl.AP.725300_TMY3.epw)
```

Comment:
This new test/example file uses the two new SPMs for different applications:
1. Outdoor air temperature-based hot water supply temperature reset
2. Return air temperature-based supply air temperature reset
3. Return air humidity ratio-based supply air maximum humidity ratio reset

Simulation shows that all cases provide the expected results but because SPMs are currently called prior to the HVAC simulation, the reporting for 2. and 3. doesn't look as smooth as for 1. (i.e. when a node non-dependent on HVAC simulation is used).

Example of results for 1.
![image](https://user-images.githubusercontent.com/5149279/152454887-02840cba-bad9-4301-9f6a-b7ea2ed4241a.png)

Example for 2.
![image](https://user-images.githubusercontent.com/5149279/152454940-5cfcdde7-cdac-458e-9dfa-e5de0064fe40.png)

Reply:
A [question was raised](https://github.com/NREL/EnergyPlus/pull/9261/files#r799089585) about the lag in the new setpoint manager controls. I don't see that as an issue. Comparing the supply setpoint and temperature for 5ZoneAirCooled and 5ZoneSystemNodeReset I see more noise in the actual supply temperature than in the setpoint. In 5ZoneAirCooled with a constant setpoint, the supply temperature is noisy, and it's very similar in 5ZoneSystemNodeReset, and the noise is small (< 0.1C).

![image](https://user-images.githubusercontent.com/4991075/153626352-6dedfc31-0df4-4f6c-ac93-5e8c46cdb435.png)
![image](https://user-images.githubusercontent.com/4991075/153626277-76e41bb1-35bc-40a2-ae02-e72296f8514e.png)

Reply:
Thanks for providing this verification :+1: